easybuild/easyconfigs/s/shift/shift-4.0-intel-2016a.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp'
+
+name = 'shift'
+version = '4.0'
+
+homepage = 'http://people.nas.nasa.gov/~kolano/projects/shift.html'
+description = """
+Shift is a framework for Self-Healing Independent File Transfer that provides high
+performance and resilience for local and remote transfers through a variety of techniques.
+"""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+sources = [SOURCE_TGZ]
+source_urls = [('http://sourceforge.net/projects/shiftc/files', 'download')]
+
+dependencies = [
+    ('Perl', '5.22.1'),
+    ('bbFTP', '3.2.1'),
+    ('mutil', '1.822.3'),
+]
+
+skipsteps = ['build']
+files_to_copy = [(['perl/shiftc', 'perl/shift-mgr', 'perl/shift-aux'], 'bin'), 'etc', (['doc/shiftc.1'], 'man/man1')]
+
+sanity_check_paths = {
+    'files': ['bin/shiftc'],
```

Comment:
yes but it's redundant information. The block should check the copied files as a default sanity check instead of specifying the same thing here twice.